kpi/serializers/v2/asset_permission_assignment.py

```diff
@@ -65,14 +65,15 @@ def get_partial_permissions(self, object_permission):
             partial_perms = asset.get_partial_perms(
                 object_permission.user_id, with_filters=True)
 
-            hyperlinked_partial_perms = []
```

Comment:
I'm going to rebase and drop this change because it's handled now by https://github.com/kobotoolbox/kpi/pull/2496/files#diff-de5fefe6d9a14b36e8d1d234855d468fR67